### ISSUE TITLE
Feature Requests: Column Customization

### ISSUE THREAD

**Issue by FanboyStudios** [mbfoss/gnethogs#11]
I would like to be able to resize the columns and pick which columns are shown or not shown.

Currently if you have a program listed with a long name, you have to scroll to the right a lot to view other columns. This could be a bit annoying. 

Other than that, the program is nice. Thanks!

Extra bonus points if you could add the ability to kill programs with a right click menu. :)
![Screenshot from 2021-05-10 10-26-37](https://user-images.githubusercontent.com/13723631/117684534-b4a87f00-b17a-11eb-9917-177214dfbfb9.png)
![Screenshot from 2021-05-10 10-26-54](https://user-images.githubusercontent.com/13723631/117684561-bd00ba00-b17a-11eb-88d0-6ac7e07fe09e.png)

**Comment by BujuArena** [comment]
Yes, the name column is extremely wide here for anything with a long command line.  If it had a maximum default width, it would make more sense.  Then I could widen it and/or hover my cursor over it to see the full command line.

**Comment by mbfoss** [comment]
This project is not longer maintained and is now archived
